rotkehlchen/tests/api/test_evm_transactions.py

```diff
@@ -6,17 +6,53 @@
 import pytest
 import requests
 
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.ethereum.modules.makerdao.sai.constants import CPT_SAI
+from rotkehlchen.constants.assets import A_ETH
 from rotkehlchen.fval import FVal
 from rotkehlchen.tests.utils.api import (
     api_url_for,
+    assert_error_response,
     assert_ok_async_response,
     assert_proper_response_with_result,
+    assert_simple_ok_response,
     wait_for_async_task,
+    wait_for_async_task_with_result,
 )
+from rotkehlchen.tests.utils.factories import make_evm_address
+from rotkehlchen.types import ChainID, ChecksumEvmAddress, Location, TimestampMS
+from rotkehlchen.utils.hexbytes import hexstring_to_bytes
 
 if TYPE_CHECKING:
     from rotkehlchen.api.server import APIServer
 
+ADDY = make_evm_address()
+
+
+def _assert_evm_transaction_in_db(
+        cursor,
```

Comment:
we don't enforce types in tests but since you have added them this is `DBCursor`